src/patient-lab-dashboard/upload-report/upload-report.resources.tsx

```diff
@@ -138,3 +147,69 @@ const removeBase64 = fileData => {
     fileData.length,
   )
 }
+
+export const bahmniEncounter = (
+  locationUuid: string,
+  patientUuid: string,
+  providerUuid: string,
+  selectedTest: LabTest,
+  selectedPendingOrder: PendingLabOrders[],
+  ac: AbortController,
+) => {
+  let requestBody: BahmniEncounteRequestType
+  if (providerUuid && providerUuid !== null) {
+    requestBody = {
+      locationUuid,
+      patientUuid,
+      encounterUuid: null,
+      visitUuid: null,
+      providers: [
+        {
+          uuid: providerUuid,
+        },
+      ],
+      visitType: 'OPD',
```

Comment:
can this be configured ?

Reply:
We can't hardcode visit type. All visit types are a configuration which implementers set. Have to have a configuration for this value.

Reply:
configuration should come from app config which is fetched initially and stored in localStorage. I will make the changes to get the visitType value from localStorage